tests/playwright/pages/wp-admin-page.js

```diff
@@ -73,15 +74,67 @@ module.exports = class WpAdminPage extends BasePage {
 	}
 
 	/**
-	 * Determine which experiments are active / inactive.
+	 * Activate / Deactivate Experiments.
 	 *
 	 * TODO: The testing environment isn't clean between tests - Use with caution!
 	 *
 	 * @param {Object} experiments - Experiments settings ( `{ experiment_id: true / false }` );
+	 * @param {string} mode        - cli (default) or ui
 	 *
 	 * @return {Promise<void>}
 	 */
-	async setExperiments( experiments = {} ) {
+	async setExperiments( experiments = {}, mode = 'cli' ) {
+		if ( 'ui' === mode.toLowerCase() ) {
+			await this._setExperimentsInUI( experiments );
+			return;
+		}
+
+		await this._setExperimentsInCLI( experiments );
+	}
+
+	/**
+	 * Set experiments using the CLI.
+	 *
+	 * @param {Object} experiments - Experiments settings ( `{ experiment_id: true / false }` );
+	 */
+	async _setExperimentsInCLI( experiments = {} ) {
+		let experimentsToEnable = [],
+			experimentsToEnableString,
+			experimentsToDisable = [],
+			experimentsToDisableString;
+
+		await parseExperiments();
+		const cli = new Terminal;
+
+		if ( experimentsToEnableString ) {
+			await cli.experiments( 'activate', experimentsToEnableString );
+		}
+		if ( experimentsToDisableString ) {
+			await cli.experiments( 'deactivate', experimentsToDisableString );
+		}
+
+		async function parseExperiments() {
+			for ( const [ id, state ] of Object.entries( experiments ) ) {
+				switch ( true ) {
+					case 'inactive' === state || 'false' === state:
+						experimentsToDisable = [ ...experimentsToDisable, id ];
+						break;
+					case 'active' === state || 'true' === state:
+						experimentsToEnable = [ ...experimentsToEnable, id ];
```

Comment:
Why not just return experiments[...experiments, id, state]

Reply:
There's a mismatch between the verbs we use for `wp elementor experiments` and verbs in use in Playwright.
parseExperiments builds a csv string to use with `wp elementor experiments`.  The idea in how I'm extending this is to keep the setExperiments() interface the same in Playwright so existing tests don't break.

I'm expecting true/false active/inactive values and in the end calling 2 cli commands like this:

`wp elementor experiments activate container,nested_elements`
`wp elementor experiments deactivate container,nested_elements`

Reply:
In my opinion, we should keep it simple: 
await wpAdmin.setExperimentsUI( { e_swiper_latest: false } );

async setExperimentsCli( experiment, isActive) {
     const terminal = new Terminal();
      const state= isActive ? 'activate' : 'deactivate'
     const cliString = 'wp elementor experiments ${state} ${experiment}'
      await terminal(cliString)
}
No need to combine two functions in one. Lets keep single responsibility principle. To refactor UI function name is not a big problem. 
But of course, the decision is up to you.